src-docs/src/views/guidelines/accessibility.js

```diff
@@ -0,0 +1,606 @@
+  </li>
+</ul>`}</EuiCodeBlock>
+
+      <h4>Developer tip</h4>
+      <p>
+        Give lists an accessible name to improve their discoverability!
+        <EuiCodeBlock language="html">{`<h1 id="a1b2c3">My favorite fruit</h1>
+<!-- Can be any heading level or even a paragraph -->
+<ul aria-labelledby="a1b2c3">...</ul>
+
+<!-- You can still provide an accessible title even if there's no visual label -->
+<ul aria-label="My favorite vegetables">...</ul>
+`}</EuiCodeBlock>
+      </p>
+      <h4>Further reading</h4>
+      <ul aria-labelledby="names further-reading">
+        <li>
+          the algorithm that figures out accessible names:
+          <br />
+          <EuiLink href="https://www.w3.org/TR/accname-1.1/#mapping_additional_nd_te">
+            Accessible Name and Description Computation
+          </EuiLink>
+        </li>
+        <li>
+          <EuiLink href="https://developer.paciellogroup.com/blog/2017/04/what-is-an-accessible-name">
+            What is an accessible name? | The Paciello Group
+          </EuiLink>
+        </li>
+        <li>
+          <EuiLink href="https://webaim.org/techniques/forms/controls">
+            WebAIM: Creating Accessible Forms - Accessible Form Controls
+          </EuiLink>
+        </li>
+        <li>
+          <EuiLink href="https://www.w3.org/WAI/tutorials/forms/labels/">
+            Labeling Controls • Forms • WAI Web Accessibility Tutorials
+          </EuiLink>
+        </li>
+        <li>
+          <EuiLink href="https://hacks.mozilla.org/2019/06/how-accessibility-trees-inform-assistive-tech">
+            How accessibility trees inform assistive tech - Mozilla Hacks
+          </EuiLink>
+        </li>
+        <li>
+          <EuiLink href="https://alistapart.com/article/semantics-to-screen-readers">
+            Semantics to Screen Readers – A List Apart
+          </EuiLink>
+        </li>
+      </ul>
+    </EuiText>
+
+    <GuideRuleTitle>Testing considerations</GuideRuleTitle>
+    <EuiSpacer size="xl" />
+    <EuiText className="guideSection__text" grow={false}>
+      <p>
+        There are a lot of aspects to accessibility and covering all the bases
+        can be a lot to keep in mind. By relying on standards we can minimize
+        the amount of special casing we have to do in code but we should still
+        be cognizant of the many modalities users might use our products.
+      </p>
+      <h3>Low-vision</h3>
+      <p>
+        While low-vision users may use many assistive technologies in tandem,
+        for this section of the guides we&apos;ll primarily talk about zooming.
+        Two ways that users may zoom the page are increasing the base font-size
+        with browser tools or using a 3rd-party magnifier (sometimes a physical
+        magnifier) to better see the screen.
+      </p>
+      <p>
+        <EuiLink href="https://www.w3.org/WAI/WCAG21/quickref/#resize-text">
+          WCAG 1.4.4
+        </EuiLink>
```

Comment:
Missing space after Link
```suggestion
        </EuiLink>{' '}
```